dbr/django_business_rules/views.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.views.generic.edit import UpdateView
+from django.views.generic.list import ListView
+
+from django_business_rules.models import BusinessRuleModel
+
+
+class BusinessRuleListView(ListView):
+    model = BusinessRuleModel
+    context_object_name = 'business_rule_list'
+    template_name = 'django_business_rules/business_rule_list.html'
+
+
+class BusinessRuleFormView(UpdateView):
+    model = BusinessRuleModel
+    fields = ('rules', )
+    template_name = 'django_business_rules/business_rule_form.html'
+
+    def get_context_data(self, **kwargs):
```

Comment:
I overridden get_context_data because I need to have access to self.object in template.
form_valid() is unnecessary.